samples/sid_end_device/src/sidewalk.c

```diff
@@ -365,11 +365,17 @@ static void state_sidewalk_run(void *o)
 
 static void state_sidewalk_exit(void *o)
 {
-	while (sys_slist_is_empty(&pending_message_list) == false) {
-		sidewalk_msg_t *message;
-		message = SYS_SLIST_CONTAINER(sys_slist_get(&pending_message_list), message, node);
+	//  returns NULL on empty list, or HEAD node.
```

Comment:
Those comments aren't much descriptive then the code itself, I think you can remove them

Reply:
removed